releasenotes/notes/networkpath-use-default-udp-port-4145a4b3700e98f4.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    For Network Traffic based paths, the default UDP port
+    traceroute port 33434, rather than the port detected by NPM,
+    is now used.
```

Comment:
```suggestion
    The default UDP port for traceroute (port 33434) is now used for Network Traffic based paths, instead of the port detected by NPM.
```

Reply:
Thanks for the suggestion!